flexget/plugins/sites/newpct.py

```diff
@@ -78,6 +78,14 @@ def parse_download_page(self, url):
                 match = torrent_id_prog.search(torrent_ids[0])
                 if match:
                     torrent_id = match.group(1)
+            if not torrent_id:
+                torrent_id_prog = re.compile('function openTorrent.*\n.*\{.*(\n.*)+window\.location\.href =\s*\".*\/(\d+).*\";')
+                torrent_ids = soup.findAll(text=torrent_id_prog)
+                log.debug('torrent ID not found, searching openTorrent script')
+                if len(torrent_ids):
```

Comment:
no need to check `len(torrent_ids)` for truthiness, `if torrent_ids` will suffice